src/mca/oob/tcp/oob_tcp_component.c

```diff
@@ -114,7 +113,7 @@ prte_oob_tcp_component_t prte_oob_tcp_component = {
             PRTE_OOB_BASE_VERSION_2_0_0,
             .mca_component_name = "tcp",
             PRTE_MCA_BASE_MAKE_VERSION(component, PRTE_MAJOR_VERSION, PRTE_MINOR_VERSION,
-                                        PRTE_RELEASE_VERSION),
+                                        PMIX_RELEASE_VERSION),
```

Comment:
Same as above, should this be `PRTE_RELEASE_VERSION`?

Reply:
Dude, once is enough! 😄